### ISSUE TITLE
Formatting fails to include define enclosed in single quotes.

### ISSUE THREAD

**Issue by 7run** [sile/efmt#105]
Thank you for providing an appropriate solution the other day.

We have noticed that constants cannot be formatted if they are enclosed in single quotes.

Input values

```erlang
-module(cat).

-define('meow-1', "Meow").
    -define('meow-2', "Meow Meow").
-define('meow-3', "Meow Meow Meow").

-export([meow/1]).

meow(1) -> ?'meow-1';
meow(2) -> ?'meow-2';
meow(3) -> ?'meow-3'.
```

The actual formatting does not seem to fail and cause an error, but the formatting does not finish and times out.

Please let me know if there is a workaround.

**Comment by sile** [comment]
Thank you for reporting this bug!
Fixed it in #106, and I will release the next version soon.

**Comment by 7run** [comment]
I was able to use the fixed version successfully :smile: 
Thank you for responding so quickly!
